src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -10,6 +10,14 @@ public function __construct($message = 'Empty values are not allowed!')
     }
 }
 
+class ObjectNotAllowedException extends \RuntimeException
+{
+    public function __construct($message = "Objects as values are not allowed!")
```

Comment:
Don't forget to update the default message. Maybe pass the value as constructor parameter, so we can dynamically build a message like `<VALUE TYPE> is not an acceptable export value.`.